docs/getting-started/README.md

```diff
@@ -1,6 +1,10 @@
-# Before you begin
+# Getting Started with a Knative "Quickstart" Environment
```

Comment:
@abrennan89 how about now? The kn CLI plugin for this will eventually be called "Quickstart" anyway 👍🏼

Reply:
Also, linked to the Admin guide for production installation